kubeflow/kpt-set.sh

```diff
@@ -67,3 +67,4 @@ kpt cfg set common/managed-storage bucket-name "${BUCKET_NAME}"
 # apps/pipelines uses specified CloudSQL and Cloud Storage bucket.
 kpt cfg set apps/pipelines cloudsql-name "${CLOUDSQL_NAME}"
 kpt cfg set apps/pipelines bucket-name "${BUCKET_NAME}"
+kpt cfg set apps/pipelines default-pipeline-root "gs://${BUCKET_NAME}/v2/artifacts"
```

Comment:
Should we define `default-pipeline-root` env var in env.sh? So user can set default pipeline root if they want. Or define `default-pipeline-root-path` which is defaulted to `v2/artifacts`. Then we can compose the `default-pipeline-root` using "gs://${BUCKET_NAME}/${default-pipeline-root-path}".

On the other hand, maybe you pin this path on purpose, happy to hear more about rational.

Reply:
My major reason is that this config is not very useful. I expect most people can use the default value. Therefore, I preferred omitting it from env vars.

Reply:
Sounds good to me for preferring default value.